packages/ove-asset-core/src/OVE.Service.Core/FileOperations/S3/S3AssetFileOperations.cs

```diff
@@ -154,49 +143,58 @@ public class S3AssetFileOperations : IAssetFileOperations {
 
         }
 
-        private async Task<List<string>> UnZipAsset(IAmazonS3 s3Client, OVEAssetModel asset, IFormFile upload) {
-            string prefixFolder = asset.StorageLocation.Split("/").FirstOrDefault() + "/";
+        public async Task<bool> UploadDirectory(string file, OVEAssetModel asset) {
+            _logger.LogInformation("about to upload directory " + file);

```

Comment:
Log should have path containing `_files` not ending with `.dzi`.

Reply:
I have refactored this method to make it clearer what's going on and that it uploads both an index file and a folder